### DISCUSSION TITLE
[Closed] because I can't read, sorry

### DISCUSSION THREAD

**Discussion by Merlinux-source** [eafer/rdrview#17] [Ideas]

**Comment by eafer** [comment]
> Imagine being able to render your simplified articles in markdown. sadly it isn't really possible to make a program which does this after the conversion (to simple text), because all the information on headers (like h1 h2 h3...) and pictures are gone.
> So Being able to export the information of the article like, which word is bold and which images are shown (and especially where) would be awesome. 

You can extract HTML instead of text using the -H option, is that what you
need?

**Reply by Merlinux-source** [reply]
Yes just what I need, as I realised this I tried to undo my mistake of writing this.

**Comment by eafer** [comment]
> Yes just what I need, as I realised this I tried to undo my mistake of writing this.

No problem, I don't mind the questions.
